.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "workbench.settings.editor": "json"
+}
```

Comment:
??

Reply:
Was manually updating my vscode setting, trying to get a Ruby formatter that works.  I'll remove.